scipy/io/tests/test_mmio.py

```diff
@@ -19,97 +19,107 @@ def setUp(self):
 
     def tearDown(self):
         shutil.rmtree(self.tmpdir)
+    
+    def test_all(self):
+        def single_test(a, info):
```

Comment:
No, but now I changed the code so that `single_test` is called directly for each test. So it is easier to see which test fails.